src/vs/workbench/contrib/terminal/common/scripts/shellIntegration-bash.sh

```diff
@@ -240,6 +251,9 @@ __vsc_command_complete() {
 		builtin printf '\e]633;D;%s\a' "$__vsc_status"
 	fi
 	__vsc_update_cwd
+
+	# if [ "$__vsc_stable" = "0" ]; then
```

Comment:
I thought 
```
if [ "$__vsc_stable" = "0" ]; 
```
 would be truthy if I'm on insiders, but testing on insiders OSS it doesnt seem to run __vsc_update_env function

Reply:
It should work as it's important this doesn't happen on stable:

https://github.com/microsoft/vscode/blob/e8184ed571509b6bc1f7752ff2216116f6194ff3/src/vs/workbench/contrib/terminal/common/scripts/shellIntegration-bash.sh#L266-L269

It works by looking at `product.json`:

https://github.com/microsoft/vscode/blob/e8184ed571509b6bc1f7752ff2216116f6194ff3/src/vs/platform/terminal/node/terminalEnvironment.ts#L144

Reply:
I just realized it wasn't working because I did not close my if statement in bash with "fi".. hah 
Now resolved via: https://github.com/microsoft/vscode/pull/237602/commits/6bc4f4669e397fd466a857375bb29d8b8b4f7cea